Bio/NeuralNetwork/Gene/Schema.py

```diff
@@ -573,7 +573,7 @@ def matches_schema(pattern, schema, ambiguity_character='*'):
 class SchemaFactory(object):
     """Generate Schema from inputs of Motifs or Signatures."""
 
-    def __init__(self, ambiguity_symbol='*'):
+    def __init__(self, ambiguity_symbol='*', maximum_tries=150):
```

Comment:
Did you find this useful in your work to be able to change the maximum number of tries?

Reply:
I made maximum_tries an input argument as I was gonna specify maximum_tries in the unit test added, but it turned out that I didn't need it for the simplest test case. However I decided to keep maximum_tries as an input argument as it could be useful for a user.